### PR TITLE
packages aarch64 u16

### PR DESCRIPTION
**Pull request by scriptnull**
https://github.com/Shippable/beta/issues/812

### CODE REVIEW

File: shippable.jobs.yml [hunk 1]
```diff
@@ -948,7 +948,28 @@ jobs:
...
+      os: Ubuntu_16.04
+      container: false
+    steps:
+      - IN: config_repo
+        switch: off
+      - IN: aarch64_u16_repo
+      - IN: ship_dh_cli_v2
+      - TASK:
+        - script: ./IN/config_repo/gitRepo/buildImage.sh u16 drydockaarch64
+      - OUT: u16_img
```
Comment (scriptnull) [inline shippable.jobs.yml]: aarch64_u16_img

File: shippable.jobs.yml [hunk 2]
```diff
@@ -948,7 +948,28 @@ jobs:
...
+      architecture: aarch64
+      os: Ubuntu_16.04
+      container: false
+    steps:
+      - IN: config_repo
+        switch: off
+      - IN: aarch64_u16_repo
+      - IN: ship_dh_cli_v2
+      - TASK:
+        - script: ./IN/config_repo/gitRepo/buildImage.sh u16 drydockaarch64
```
Comment (shrivara) [inline shippable.jobs.yml]: Not quite sure how this works. I think we pushd into the context of the folder. So, shouldn't this be aarch64_u16?

File: shippable.resources.yml [hunk 3]
```diff
@@ -861,7 +861,28 @@ resources:
...
+    integration: ric03uec-github
+    pointer:
+      sourceName: dry-dock-aarch64/u16
+      branch: master
 
+  - name: aarch64_u16_img
+    type: image
+    integration: shipDH
+    pointer:
+      sourceName: "drydockaarch64/u16"
```
Comment (shrivara) [inline shippable.resources.yml]: The quotes around this are unnecessary.

File: shippable.jobs.yml [hunk 4]
```diff
@@ -948,7 +948,28 @@ jobs:
...
+      - IN: config_repo
+        switch: off
+      - IN: aarch64_u16_repo
+      - IN: ship_dh_cli_v2
+      - TASK:
+        - script: ./IN/config_repo/gitRepo/buildImage.sh aarch64_u16 drydockaarch64 u16
+      - OUT: aarch64_u16_img
+
+  ###########################################
+  # START Drydockaarch64 image builds
```
Comment (scriptnull) [inline shippable.jobs.yml]: END

**Pull request merged**